src/pages/components/loading/usage.mdx

```diff
@@ -41,23 +44,290 @@ and help to notify users that loading is underway.
+experiences within a specific UI element, particularly in inline loading
+scenarios. It works well for quick-loading tasks and can be placed within or
+near the triggering element—such as a
+[button](/components/button/usage/#loading)—to indicate that a process is
+underway. Common examples include saving after a button click, uploading a file,
+or placing the indicator beside a text label to communicate loading status. For
+detailed guidance, refer to the inline loading component
+[Usage](/components/inline-loading/usage/) tab.
+
+<Row>
+
+<Column colLg={8}>
+<GifPlayer color='light'>
+
+![Example of the large loading component in a UI](/images/modal-error.gif)
+
+![Example of the large loading component in a UI](/images/modal-error.png)
+
+</GifPlayer>
+</Column>
+</Row>
+
+<Caption>Example of the small loading component in a UI</Caption>
+
+### Placement
+
+#### Large loading placement
+
+The large loading indicator is commonly used for full-screen and component-level
+loading scenarios.
+
+For full-screen loading, place the indicator at the center of the viewport and
+apply a semi-transparent overlay to signal that the entire interface is
+inactive.
+
+<Row>
+<Column colLg={8}>
+
+![Example of large loading indicator placement for full-screen loading with overlay](images/loading-usage-placement-large-full-screen.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Example of large loading indicator placement for full-screen loading with
+  overlay
+</Caption>
+
+When used within a component—such as a modal, tile, or side panel—center the
+indicator in that space and use an overlay to prevent interaction with the
+content while it is loading.
+
+<Row>
+<Column colLg={8}>
+
+![Example of large loading indicator placement for component-level loading with overlay](images/loading-usage-placement-large-component-level.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Example of large loading indicator placement for component-level loading with
+  overlay
+</Caption>
+
+#### Small loading placement
+
+The small loading indicator is ideal for inline or localized loading within
+elements like buttons or compact content areas. It should appear inline with or
+adjacent to the related action, without using overlays. During the loading
+state, any related action buttons—such as “Cancel”—should be disabled. For more
+information about inline loading, refer to the inline loading component
+[Usage](/components/inline-loading/usage/) tab.
+
+<Row>
+<Column colLg={8}>
+
+![Example of small loading indicator placement for inline loading](images/loading-usage-placement-small-inline.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Example of small loading indicator placement for inline loading
+</Caption>
+
+## Content
+
+### Main elements
+
+The loading indicator can be paired with optional text to provide additional
+context during a loading state.
+
+For **large loading** indicators, adding a brief status message below the
```

Comment:
I am going to suggest rewording this sentence as the following. You do not need to semibold in this sentence:

For large loading indicators, adding a brief status message below the indicator, such as “Loading data…” can help set users’ expectations. The label text is not included by default in the component but can be customized to include it if needed.